test/Spec.hs

```diff
@@ -450,6 +451,17 @@ statefulTests =
                     ["default"]
          `shouldReturn` Just "12"
 
+doctypeTests :: SpecWith LarcenyHspecState
+doctypeTests = do
+  describe "doctypes" $ do
+    it "should render blank doctypes" $ do
+      "<doctype />" `shouldRenderM` "<!DOCTYPE html>"
```

Comment:
should we insert doctypes even when they're not included? i guess we're not really sure we're outputting valid html5 all the time.